parm/archive/gefs_arcdir.yaml.j2

```diff
@@ -21,7 +21,7 @@
 {% if RUN == "gefs" %}
     {% set ensstat_files = [] %}
     {% if path_exists(COMIN_ATMOS_ENSSTAT_1p00) %}
-        {% for fhr in range(FHMIN_GFS, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
+        {% for fhr in range(6, FHMAX_GFS + FHOUT_GFS, FHOUT_GFS) %}
```

Comment:
Should this really be hard-coded?

Reply:
@EricSinsky-NOAA do you have a recommendation? This will be changed with the archiving PR.

Reply:
@NeilBarton-NOAA Please see my review comments below. It turns out `parm/archive/gefs_arcdir.yaml.j2` is not being modified in Anton's current PR, so we might need to make that correction here.